lang/cs/Org.Apache.REEF.Bridge.Client/GrpcGeneratedCode/ClientProtocol.cs

```diff
@@ -0,0 +1,1210 @@
+// Generated by the protocol buffer compiler.  DO NOT EDIT!
+// source: ClientProtocol.proto
+#pragma warning disable 1591, 0612, 3021
+#region Designer generated code
+
+using pb = global::Google.Protobuf;
+using pbc = global::Google.Protobuf.Collections;
+using pbr = global::Google.Protobuf.Reflection;
+using scg = global::System.Collections.Generic;
+namespace Org.Apache.REEF.Bridge.Proto {
+
+  /// <summary>Holder for reflection information generated from ClientProtocol.proto</summary>
+  public static partial class ClientProtocolReflection {
```

Comment:
>ClientProtocolReflection [](start = 30, length = 24)

Are any of those new fiels and proto that you have created testable?  how would you know if this work?  SHould there be any unit/integration tests?

Reply:
The proto files are from Tyson. I am going to pull his request later today if all of the tests pass so I will not need to include any of his files in future pull request.

Reply:
We need to add integration tests that one of the reasons why we are currently pulling into the REEF-335 branch and not master. However, this allows us to currently share code.